src/demos/_assets/nav.html

```diff
@@ -0,0 +1,46 @@
+<template>
```

Comment:
We could call this file `demo-template.html` and `demo-template.ts` if that makes more sense.